### PR TITLE
loadtest assets for ansible-like sessions test

### PR DESCRIPTION
**Pull request by espadolini**

### CODE REVIEW

**Review by strideynet — APPROVED** [review]

File: assets/loadtest/ansible-like/tbot_install.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+cd $( dirname -- ${0} )
+
+sudo loginctl enable-linger ${USER}
+
+curl https://goteleport.com/static/install.sh | sudo bash -s 15.4.3 enterprise || exit $?
```
[the remark this answers is not in the document]
Comment (espadolini) [reply assets/loadtest/ansible-like/tbot_install.sh:6]: If you need to check the script for which envvar you're supposed to set to set the Teleport version, you might as well set the version there.

File: assets/loadtest/ansible-like/proxy_templates.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+proxy_templates:
+  - template: "^(.*).xltenant.teleport.sh:[0-9]+$" # <nodename>.<clustername>:<port>
+    query: 'contains(split(labels.NODENAME, ","), "$1")'
+    # query: 'labels.NODENAME == "$1"'
```
[the remark this answers is not in the document]
Comment (espadolini) [reply assets/loadtest/ansible-like/proxy_templates.yaml:4]: I haven't tested fuzzy search yet but we probably should.

File: assets/loadtest/ansible-like/tbot_install.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (espadolini) [reply assets/loadtest/ansible-like/tbot_install.sh:6]: USER is set by default, although we could even omit it to also have it default to the current user.

File: assets/loadtest/ansible-like/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Ansible-like OpenSSH sessions load test
+
+This setup is designed to be ran from the home directory of a VM (the default working directory for a systemd user service) and the xltenant.teleport.sh service; the proxy public address and cluster name should be changed in `gen_inventory.sh`, `proxy_templates.yaml` and `tbot.yaml` for use in a different cluster. It requires openssh, jq, xargs, and dumb-init.
```
[the remark this answers is not in the document]
Comment (espadolini) [reply assets/loadtest/ansible-like/README.md]: The files that require changing are those listed in the paragraph. Should I change that to a fake cluster name?

[the remark this answers is not in the document]
Comment (espadolini) [reply assets/loadtest/ansible-like/README.md]: Got rid of the specific cloud cluster mentions, as they're liable to get outdated.

**Review by rosstimothy — APPROVED** [review]

**Pull request merged**